packages/discord.js/src/structures/AttachmentBuilder.js

```diff
@@ -108,3 +108,9 @@ module.exports = AttachmentBuilder;
  * @external APIAttachment
  * @see {@link https://discord.com/developers/docs/resources/channel#attachment-object}
  */
+
+/**
+ * @typedef {Object} AttachmentData
+ * @property {name} [string] The name of the attachment
+ * @property {description} [string] The description of the attachment
```

Comment:
```suggestion
 * @property {string} [name] The name of the attachment
 * @property {string} [description] The description of the attachment
```

Reply:
🤦🏻 Whoops, I was a bit in a rush when I wrote that 😅